src/styles/theme.css

```diff
@@ -2,6 +2,11 @@
 @tailwind components;
 @tailwind utilities;
 
+@font-face {
+  font-family: 'Cooper Black';
+  src: url('../assets/COOPBL.woff2') format('woff2');
```

Comment:
Only tweak that comes to mind (and I don't mind adding myself) is maybe trying to add [`font-display: swap`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display#values) as I _think_ it makes the transition or ["popping" in of fonts](https://developer.chrome.com/blog/font-display/#which-font-display-is-right-for-you) a little more subtle.

Not sure if you have any thoughts / experience.